packages/elements-core/src/index.ts

```diff
@@ -0,0 +1,32 @@
+export { Docs, ParsedDocs } from './components/Docs';
```

Comment:
perhaps it's just me overcomplicating things as always, but I would either:

- separate with comments which stuff is used in "3rd party" apps (platform, studio etc.)
- or even place them in separate file (`@stoplight/elements/protected` or something)

Because currently we don't know which changes affect only "our" repos and which affect 3rd party users.

Might make versioning easier? 🤔

Reply:
Yeah, some kind of annotation might be helpful

Reply:
We agreed to tackle this separately.